README.md

```diff
@@ -14,6 +14,7 @@
 
 ## Introduction
 
+Welcome to AXLearn, a powerful deep learning library built on top of JAX and XLA. AXLearn is designed to support the development of large-scale deep learning models and offers a unique approach to tackling the software engineering challenges that arise during model development and maintenance.
```

Comment:
The claim about "unique" is not entirely accurate. The object-oriented approach was adopted at least by lingvo and paxml.